ouroboros-consensus/src/Ouroboros/Consensus/Protocol/PBFT.hs

```diff
@@ -310,8 +302,8 @@ instance PBftCrypto c => ConsensusProtocol (PBft c) where
                       slot
                       (TickedPBftState (TickedPBftLedgerView dms) state) =
       case toValidate of
```

Comment:
This is a nice simplification to see. EBBs were only tracked by PBFT state to enable rewind. No more rewind means no more EBB code 🙌

Reply:
Indeed, this made me very happy and was a sign to me that this is a good refactoring :slightly_smiling_face:.